jsk_apc2016_common/data/object_list.yaml

```diff
@@ -0,0 +1,39 @@
+- i_am_a_bunny_book
```

Comment:
i found out that there is no object name list for apc2016, and it  should be useful.